src/main/webapp/app/exercises/programming/manage/update/update-components/programming-exercise-information.component.html

```diff
@@ -45,6 +45,23 @@
             }
         }
     </div>
+    @if (!programmingExerciseCreationConfig.isExamMode) {
```

Comment:
Only moved up in the file to be more consistent between the edit and details view